v23.1/show-ranges.md

```diff
@@ -25,96 +31,1545 @@ To use the `SHOW RANGES` statement, a user must either be a member of the [`admi
+`table_start_key` | The start key of the first [range](architecture/overview.html#architecture-range) that holds data for this table.
+`table_end_key` | The end key of the last [range](architecture/overview.html#architecture-range) that holds data for this table.
+`index_name` | The name of the [index](indexes.html) this [range](architecture/overview.html#architecture-range) holds data for.
+`index_id` | The internal ID of the [index](indexes.html) this [range](architecture/overview.html#architecture-range) holds data for.
+`index_start_key` | The start key of the first [range](architecture/overview.html#architecture-range) of [index](indexes.html) data.
+`index_end_key` | The end key of the last [range](architecture/overview.html#architecture-range) of [index](indexes.html) data.
 
 {{site.data.alerts.callout_info}}
 If both `start_key` and `end_key` show `NULL`, the range is empty and has no splits.
 {{site.data.alerts.end}}
 
 ## Examples
 
+{{site.data.alerts.callout_info}}
+{% include {{page.version.version}}/sql/show-ranges-output-deprecation-notice.md %}
+{{site.data.alerts.end}}
+
 {% include {{page.version.version}}/sql/movr-statements-geo-partitioned-replicas.md %}
 
-### Show ranges for a table (primary index)
+### Show ranges for a database
+
+- [Show ranges for a database (without options)](#show-ranges-for-a-database-without-options)
+- [Show ranges for a database (with tables)](#show-ranges-for-a-database-with-tables)
+- [Show ranges for a database (with indexes)](#show-ranges-for-a-database-with-indexes)
+- [Show ranges for a database (with details)](#show-ranges-for-a-database-with-details)
+- [Show ranges for a database (with keys)](#show-ranges-for-a-database-with-keys)
+- [Show ranges for a database (with explain)](#show-ranges-for-a-database-with-explain)
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW DATABASES;
+~~~
+
+~~~
+  database_name | owner | primary_region | secondary_region | regions | survival_goal
+----------------+-------+----------------+------------------+---------+----------------
+  defaultdb     | root  | NULL           | NULL             | {}      | NULL
+  movr          | demo  | NULL           | NULL             | {}      | NULL
+  postgres      | root  | NULL           | NULL             | {}      | NULL
+  system        | node  | NULL           | NULL             | {}      | NULL
+(4 rows)
+~~~
+
+#### Show ranges for a database (without options)
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW RANGES FROM DATABASE movr;
+~~~
+
+~~~
+                                           start_key                                           |                                           end_key                                            | range_id | replicas |                                 replica_localities                                 | voting_replicas | non_voting_replicas | learner_replicas |    split_enforced_until
```

Comment:
It's not valuable in the doc page to include the entire output. You can elide all the rows except for the first 2 and the last one and replace them with an ellipsis.

Reply:
OK - did this for all examples on this page